server/perfkit/explorer/handlers/base.py

```diff
@@ -90,6 +90,10 @@ def RenderHtml(self, template_file, template_values, status=200):
     template_values['static_dir'] = ('/_static/%s' %
                                      os.environ['CURRENT_VERSION_ID'])
     template_values['env'] = self.env
+    if 'PKE_DEBUG' in os.environ:
```

Comment:
Recommend allowing this parameter to be used in the URL as well, so that it can be used on deployments.